radixdlt-core/docker/Dockerfile.core

```diff
@@ -1,11 +1,6 @@
-FROM ubuntu:21.04
+FROM ubuntu:22.04
 MAINTAINER radixdlt <devops@radixdlt.com>
 
-#https://issueexplorer.com/issue/tianon/docker-brew-ubuntu-core/207
```

Comment:
Based on this link, if people aren't running:
* Docker version 19.03.9 or newer
* libseccomp version 2.4.2 or newer

Then building this image will fail. So I think it's worth at least flagging this minimum version in the README

Reply:
Will the removed lines make it work? If so, not sure if they should be removed.

Reply:
Added `Requirements` section to relevant README.me.

Reply:
Not sure if it is true, the comment in the link says:  "if using --security-opt seccomp=unconfined makes it work, you'll need to update your host to...". We are not using `--security-opt seccomp=unconfined`.

Reply:
Seems we were using the solution in the last comment. If it is not necessary anymore, I guess we can just remove it and no version requirements are needed, but it might be good to test it with a docker version before 19.03.9.

Reply:
I think it's fine to remove the version requirement. If there will be any problem in the future, we can revisit it.

Reply:
Removed.